password_security/models/res_users.py

```diff
@@ -97,6 +97,10 @@ def _password_has_expired(self):
         self.ensure_one()
         if not self.password_write_date:
             return True
+
+        if not self.company_id.password_expiration:
```

Comment:
Where is this field defined?

Reply:
The field is defined in password_security/models/res_company.py

```
class ResCompany(models.Model):
    _inherit = 'res.company'

    password_expiration = fields.Integer(
        'Days',
        default=60,
        help='How many days until passwords expire',
    )
    ...
```

Reply:
OK, seen. I have mixed modules.

Reply:
Great! Thank you so much! :)